src/openforms/translations/tests/test_api.py

```diff
@@ -0,0 +1,131 @@
+from collections import namedtuple
+
+from django.test import override_settings
+from django.urls import reverse
+from django.utils.translation import gettext as _
+
+from hypothesis import given, settings, strategies as st
+
+# from hypothesis.extra.django import TestCase
+from rest_framework.test import APISimpleTestCase
+
+Language = namedtuple("Language", "conf local")
+
+
+class I18NAPITests(APISimpleTestCase):
+    databases = ["default"]
```

Comment:
please don't override settings/configurations that are already the default. it's highly unlikely we'll ever have a multi-db setup